src/routes/namespaces/[namespace]/workflows/index@root.svelte

```diff
@@ -3,10 +3,13 @@
 
   import Workflows from '$lib/pages/workflows.svelte';
   import PageTitle from '$lib/components/page-title.svelte';
+  import { temporalVersion } from '$lib/stores/versions';
+  import { isVersionNewer } from '$lib/utilities/version-check';
 </script>
 
 <PageTitle
   title={`Workflows | ${$page.params.namespace}`}
   url={$page.url.href}
 />
-<Workflows />
+
+<Workflows bulkActionsEnabled={isVersionNewer($temporalVersion, '1.18.0')} />
```

Comment:
@Alex-Tideman do we need to just set this to `false` for this PR? I think we're good to release this properly once ui-server is updated. I think it's a fairly polished experience, except maybe for large batches.

Reply:
I think we should set this to false on the page itself (right now export let bulkActionsEnabled = true). Because this should get deployed with the ui-server change when it's merged, but if we update package.json version, cloud will get this but the ui-server won't have been deployed yet